lab_5_scraper/scraper.py

```diff
@@ -206,6 +310,10 @@ def prepare_environment(base_path: Union[pathlib.Path, str]) -> None:
     Args:
         base_path (Union[pathlib.Path, str]): Path where articles stores
     """
+    if base_path.exists() and base_path.is_dir():
```

Comment:
you can just remove all directory at once with shutil, not need to iterate over the directory contents